go/package/download.php

```diff
@@ -26,20 +27,28 @@
 
   class PackageDownloadPage {
 
-    public static function redirect_to_file($id, $version, $platform, $tier, $bcp47, $update, $ga_cookie) {
+    public static function redirect_to_file($type, $id, $version, $platform, $tier, $bcp47, $update, $ga_cookie) {
+      if(empty($type)) {
+        $type = 'keyboard';
+      }
+
       if(empty($id)) {
         JsonApiFailure::InvalidParameters("id, version");
       }
 
+      if($type !== 'keyboard' && $type !== 'model') {
```

Comment:
Just wondering if this is intentional to validate $id (2nd parameter) before $type (1st parameter)?
Otherwise, I might move this to l. 34.